docs/02-get-started/02-deploy-expose-function.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Deploy and expose a Function
+---
+
+Now that you've installed Kyma, let's deploy your first Function. We'll call it `hello-world`.
+
+## Create a function
+
+First, let's create the Function and apply it.
+
+<div tabs name="Deploy a Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+kyma init function --name hello-world
+kyma apply function
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In Kyma Dashboard, go to the `default` Namespace.
+2. Go to **Workloads** > **Functions**.
+3. Click on **Create Function +**.
+4. Name the Function `hello-world` and click **Create**.
+  </details>
+</div>
+
+
+### Verify the Function deployment
+
+Now let's make sure that the Function has been deployed successfully. 
+
+<div tabs name="Verify the Function deployment" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+kubectl get functions hello-world 
+```
+
+The operation was successful if the statuses for **CONFIGURED**, **BUILT**, and **RUNNING** are `True`.
+
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+The operation was successful if the Function **Status** changed from `DEPLOYING` to `RUNNING`.
+
+> **NOTE:** You might need to wait a few seconds for the status to change.
+  </details>
+</div>
+
+## Expose the Function
+
+After we've got our `hello-world` Function deployed, we might want to expose it outside our cluster so that it's available for other external services.
+
+> **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always [secure the workload](../03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md) you expose.
+
+First, let's create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) for the Function.
+
+<div tabs name="Expose the Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: hello-world
+    namespace: default
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - accessStrategies:
+        - config: {}
+          handler: allow
+        methods:
+          - GET
+          - POST
+          - PUT
+          - PATCH
+          - DELETE
+          - HEAD
+        path: /.*
+    service:
+      host: hello-world.$CLUSTER_DOMAIN
+      name: hello-world
+      port: 80
+EOF
+```
+
```

Comment:
I get the following message :'(

![image](https://user-images.githubusercontent.com/76950046/133641689-1b93f046-2db9-4f13-9c2c-a9ead8689e9c.png)

Reply:
*facepalm
I had to replace `$CLUSTERDOMAIN` with` local.kyma.dev`